### PR TITLE
Ultiduo 18 Ruleset Update

### PR DESCRIPTION
**Pull request by Chilcott**

### CODE REVIEW

File: docs/rules/ultiduo.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Ultiduo Ruleset
-This ruleset was made for Ultiduo 18/future Ultiduo competitions run by ozfortress.
+This ruleset was last updated for Ultiduo 18 - please consult with the ozfortress league admin team for any future Ultiduo competitions in case there has been any updates.
```
Comment (coreobs) [inline docs/rules/ultiduo.md]: The "last updated" note at the top isn't needed, our docs show when they were last updated. That section was mainly to note that this ruleset was added for UD18 and beyond.

File: docs/rules/ultiduo.md [hunk 2]
```diff
@@ -24,10 +24,19 @@ Failure to observe weapon whitelisting will result in league penalties. For Ulti
 - **Medic**
     - Syringe Gun & Syringe Gun Reskins
     - Crusader's Crossbow
+ 
+### 1.3. Weapon Pickups
+Weapons dropped on the ground by dead players are **allowed** to be picked up by surviving players - including the medigun.
```
Comment (coreobs) [inline docs/rules/ultiduo.md]: Would read better as "Dropped weapons can be picked up - including any medigun."

File: docs/rules/ultiduo.md [hunk 3]
```diff
@@ -24,10 +24,19 @@ Failure to observe weapon whitelisting will result in league penalties. For Ulti
...
 
-### 1.3. Map Scoring
+### 1.4. Map Scoring
 In Ultiduo, all maps are King of the Hill (KOTH) maps, and follow a best of one win condition with a five minute team clock. Different KOTH maps may vary slightly on overtime timers, locked and unlocked timers (when the point is not able to be captured at the start and when it unlocks) as these are built into the map.
 
+### 1.5 Tie Breakers
+As the KOTH map standard, each map will be placed until a team has scored 3 rounds. No Tie Breaker should be needed for individual maps.
```
Comment (coreobs) [inline docs/rules/ultiduo.md]: Tiebreakers should discuss Citadel Table Tiebreakers - which we haven't decided on yet. Map tiebreakers won't be needed as the config does not allow for ties.

File: docs/rules/ultiduo.md [hunk 4]
```diff
@@ -24,10 +24,19 @@ Failure to observe weapon whitelisting will result in league penalties. For Ulti
...
 
-### 1.3. Map Scoring
+### 1.4. Map Scoring
 In Ultiduo, all maps are King of the Hill (KOTH) maps, and follow a best of one win condition with a five minute team clock. Different KOTH maps may vary slightly on overtime timers, locked and unlocked timers (when the point is not able to be captured at the start and when it unlocks) as these are built into the map.
 
+### 1.5 Tie Breakers
+As the KOTH map standard, each map will be placed until a team has scored 3 rounds. No Tie Breaker should be needed for individual maps. 
+
+### 1.6 Scheduling 
+Due to the time constraints surrounding Ultiduo 18, once the string is sent in the Team Captain Discord, both players must immediately join the server. Failure to have 2 rostered players readied up in the server within five minutes of start time will resolve in a forfeit loss.
```
Comment (coreobs) [inline docs/rules/ultiduo.md]: "result" instead of "resolve".

File: docs/rules/ultiduo.md [hunk 5]
```diff
@@ -38,10 +47,50 @@ For playoff matches following a best of three (BO3) format, the **higher seeded
 
 The **higher seeded team** is **Team A**. **Maps must be played in the order in which they were picked.**
 
+Using the Captain Discord Match Thread channel
```
Comment (coreobs) [inline docs/rules/ultiduo.md]: Match pick/bans for Ultiduo can and should be done using the pick/bans system on Citadel. The only reason it's not doable for our Sixes seasonal league is we have to add ~11 maps to the league page, but not all divisions are allowed to play all maps, hence why we did it via discord for weekly vetos.

File: docs/rules/ultiduo.md [hunk 6]
```diff
@@ -38,10 +47,50 @@ For playoff matches following a best of three (BO3) format, the **higher seeded
...
+The match may only be unpaused by the team who initiated the pause, with confirmation from the opposing teams. Unpausing initiates a five-second countdown.
+
+### 3.1. Pause Limits
+Each team is allowed two pauses per match, with a combined total time limit of five minutes. 
+
+### 3.2. Tactical Pauses
+Tactical pauses are allowed but contribute to the pause limits described above. They should not be used to disrupt ongoing plays.
+
+### 3.3. Pause Command
+To pause a match, teams must open their console and send the command: pause. The same command can be sent to unpause the match.
```
Comment (coreobs) [inline docs/rules/ultiduo.md]: Wrap pause in `` tags

File: docs/rules/ultiduo.md [hunk 7]
```diff
@@ -38,10 +47,50 @@ For playoff matches following a best of three (BO3) format, the **higher seeded
...
+
+### 3.4. Repause Command
+To allow a player to rejoin the server during a pause, use the command: repause. No warning is required to use the repause command.
```
Comment (coreobs) [inline docs/rules/ultiduo.md]: Same as above for repause

File: docs/rules/ultiduo.md [hunk 8]
```diff
@@ -38,10 +47,50 @@ For playoff matches following a best of three (BO3) format, the **higher seeded
...
+
+### 3.5. Abuse of Pause function
+If there are any complications or a team is breaching any of the pause rules, a tournament administration must be contacted **immediately.**
```
Comment (coreobs) [inline docs/rules/ultiduo.md]: "a tournament administration" should be "a tournament administrator"

File: docs/rules/ultiduo.md [hunk 9]
```diff
@@ -38,10 +47,50 @@ For playoff matches following a best of three (BO3) format, the **higher seeded
...
+
+## 4. Player Eligibility
+### 4.1. Ping Requirements //make this not 3 to avoid crossing over with global ruleset numbering conflictions
+ Ultiduo will not follow our Global Ruleset Ping or Out of Region rules. Out-of-Region Players intending to join a roster, or play as a mercenary, must meet the in-game ping limit of 120. If you are unsure whether you meet this requirement, prior to the tournament starting, you should contact a League Administrator to undergo a ping check. It is the responsibility of Out-of-Region Players to ensure that they satisfy this ping limit before the beginning of the tournament.
```
Comment (coreobs) [inline docs/rules/ultiduo.md]: "prior to the tournament starting, you should contact a League Administrator to undergo a ping check.", remove the comma between starting and you should

"prior to the tournament starting you should contact a League Administrator to undergo a ping check."

Also, ping limit should be at least 150.

**Review by coreobs — CHANGES_REQUESTED** [review]
See single comments for change requests

File: docs/rules/ultiduo.md (hunk 3 above)
Comment (Chilcott) [reply docs/rules/ultiduo.md]: changed wording to allow to push - still needs work

**Review by coreobs — APPROVED** [review]

**Pull request merged**